jupyterhub/apihandlers/users.py

```diff
@@ -209,14 +218,16 @@ def post(self, name):
             raise web.HTTPError(400, "%s's server is not running" % name)
         self.set_server_cookie(user)
         # a service can also ask for a user cookie
-        # this code prevent to raise an error
+        # this code prevents to raise an error
+        # cause service doesn't have 'other_user_cookies'
         if getattr(current, 'other_user_cookies', None) is not None:
             current.other_user_cookies.add(name)
 
 
 default_handlers = [
     (r"/api/users", UserListAPIHandler),
     (r"/api/users/([^/]+)", UserAPIHandler),
-    (r"/api/users/([^/]+)/server", UserServerAPIHandler),
+    (r"/api/users/([^/]+)/servers", UserCreateServerAPIHandler),
```

Comment:
For future: we'll probably want to keep `/server` endpoints for the default server. That can be restored in a new PR, though.

We'll probably also want to keep the notion of a server with no name (`name=''`), with no prefix, since most of the time server-names won't be used.